connector/mqtt/vernemq/broker/src/dojot_disconnect_plugin/src/dojot_disconnect_plugin_sup.erl

```diff
@@ -11,11 +11,14 @@
 %% Helper macro for declaring children of supervisor
 -define(CHILD(I, Type), {I, {I, start_link, []}, permanent, 5000, Type, [I]}).
 
+-define(DISCONNECT_ETS_TABLE, disconnect_ets_table).
+
 %% ===================================================================
 %% API functions
 %% ===================================================================
 
 start_link() ->
+    ets:new(disconnect_ets_table, [public, named_table]),
```

Comment:
I think the better approach is to spawn a new process to manage the ETS, but for now I think it is OK create the ETS in the supervisor process.